### ISSUE TITLE
suggestion : prompt operator/presenter that the room is locked

### ISSUE THREAD

**Issue by Miroslav36** [jech/galene#135]
When a room is locked by default (autolock), the operator must unlock the room in order to allow other participants to connect.

problem:
When reaching a room with a direct link, the operator/presenter entering the room can forget to unlock. To my knowledge there is no easy way to check if a room is locked, especially if the room is not public.

Suggestion : 
When the operator is connected to a room, display a permanent information to operators (and maybe presenters) about the locked state of the room. This could be placed next to the "enable"/"mute"/"share screen" buttons for example.

**Comment by jech** [comment]
Right.  I'm looking for a useful place to put a global menu (similar to the ones used for user management) and a status display.  Any suggestions welcome.

**Comment by jech** [comment]
I was looking for something way too complicated.  Fixed in 4053d9c.

**Comment by Miroslav36** [comment]
Thank you
